appservice/src/SiteClient.ts

```diff
@@ -60,15 +61,19 @@ export class SiteClient {
         this.kind = site.kind;
         this.initialState = site.state;
         this.isFunctionApp = site.kind && site.kind.includes('functionapp');
+        this.isLinuxConsumptionPlan = site.kind && site.kind.includes('functionapp,linux,container');
```

Comment:
So I don't think this check is correct. To check if it's linux it would just be:
```
site.kind && site.kind.includes('linux')
```

And to check if it's consumption, you'd have to check if the plan's sku was "Dynamic"